src/manage_sql_nvts.c

```diff
@@ -1302,6 +1302,14 @@ update_nvts_from_vts (entity_t *get_vts_response,
   entity_t vts, vt;
   entities_t children;
   GList *preferences;
+  int count_modified_vts = 0;
```

Comment:
Would like to see these variables initialised after the declaration section, like
the other variables in the block.  And would like the two int variables on one
line like the entity_t variables.